connection.go

```diff
@@ -421,6 +443,16 @@ func Connect(addr string, opts Opts) (conn *Connection, err error) {
 		}
 	}
 
+	// Subscribe shutdown event to process graceful shutdown.
```

Comment:
At this point, we don't always have a connected connection (and it's generally ok):

https://github.com/tarantool/go-tarantool/issues/136#issuecomment-1172865773

Reply:
Well... maybe we should add the flag like `opts.SkipSchema` one to solve both this and https://github.com/tarantool/go-tarantool/pull/249#discussion_r1056936480 ?

Reply:
You could create the watcher here:

https://github.com/tarantool/go-tarantool/blob/master/connection.go#L595

Reply:
Handling it a single time on first initialization is much more simpler that creating one on `dial` triggered on each reconnection.

Reply:
> Handling it a single time on first initialization is much more simpler that creating one on `dial` triggered on each reconnect.

Not when we need to add non-obvious options. Moreover, the watcher can be created once:

```Go
if conn.shutdownWatcher != nil {
}
```

Reply:
Ok, moved